cloudos/clos.py

```diff
@@ -340,65 +340,54 @@ def get_curated_workflow_list(self, workspace_id, get_all=True, page=1, verify=T
         r : list
             A list of dicts, each corresponding to a workflow.
         """
-        data = {"search": "",
-                "page": page,
-                "filters": [
-                    [
-                        {
-                            "isPredefined": True,
-                            "isCurated": True,
-                            "isFeatured": False,
-                            "isModule": False
-                        },
-                        {
-                            "isPredefined": True,
-                            "isCurated": False,
-                            "isFeatured": False,
-                            "isModule": False
-                        },
-                        {
-                            "isPredefined": True,
-                            "isCurated": True,
-                            "isFeatured": True,
-                            "isModule": False
-                        }
-                    ]
-                 ]
-                }
         headers = {
             "Content-type": "application/json",
             "apikey": self.apikey
         }
-        r = retry_requests_post("{}/api/v1/workflows/getByType?teamId={}".format(self.cloudos_url,
-                                                                           workspace_id),
-                                json=data, headers=headers, verify=verify)
+        r = retry_requests_get(
+            "{}/api/v3/workflows?search=&groups[]=curated&page={}&teamId={}".format(
```

Comment:
So `isCurated:true, isFeatured:false` is now group = curated
`isCurated:true, isFeatured:true` is now group = featured
`isPredefined:true` is now group = predefined

You can fetch all 3 groups here to keep previously functionality